src/sestest/Program.cs

```diff
@@ -36,12 +31,13 @@ public static int Main(string[] args)
             {
                 // Logging is ready for use, can try other things
 
-                exitCode = parseResult.MapResult(
+                exitCode = await parseResult.MapResult(
                     (GenerateCommandLine commandLine) => RunCommand(Generate, commandLine),
                     (ServerCommandLine commandLine) => RunCommand(Serve, commandLine),
                     (ListCertificatesCommandLine commandLine) => RunCommand(ListCertificates, commandLine),
                     (FindCertificateCommandLine commandLine) => RunCommand(FindCertificate, commandLine),
-                    errors => 1);
+                    (VerifyCommandLine commandLine) => RunCommand(Submit, commandLine),
+                    errors => Task.FromResult(-1));
```

Comment:
There's a `ResultCodes.Failed` that is `-1` below. Is that in-band for this value? 🙂

Reply:
Yes; fixed this and a couple of others that I missed.